File: Easy/Java/495. Teemo Attacking.java
class Solution {
    public int findPoisonedDuration(int[] timeSeries, int duration) {
        int sec = duration;

        for(int i = 1; i < timeSeries.length; i++){
            sec += Math.min(duration, timeSeries[i] - timeSeries[i - 1]);
        }

        return sec;
    }
}

File: Easy/JavaScript/495. Teemo Attacking.js
var findPoisonedDuration = function(timeSeries, duration) {
    let sec = duration;

    for(let i = 1; i < timeSeries.length; i++){
        sec += Math.min(duration, timeSeries[i] - timeSeries[i - 1]);
    }

    return sec;
};
